appveyor.yml

```diff
@@ -0,0 +1,59 @@
+# DO NOT CHANGE the "init" and "install" sections below
+
+# Download script file from GitHub
+init:
+  ps: |
+        $ErrorActionPreference = "Stop"
+        Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
+        Import-Module '..\appveyor-tool.ps1'
+
+install:
+  ps: Bootstrap
+
+cache:
+  - C:\RLibrary
+
+environment:
+  NOT_CRAN: flase
```

Comment:
Looks like a typo here. Should be "false"?

Reply:
I was trying to test `NOT_CRAN: false` and `NOT_CRAN: True`. I did not see any change in the way the checks are tested. I may be missing something on how the that tag is used.

Reply:
I have set it back to `NOT_CRAN: true`